### PR TITLE
Forfeit confirmation

### PR DESCRIPTION
**Pull request by mbrandonw**
Ask the player to confirm they want to forfeit.

### CODE REVIEW

File: Tests/GameCoreTests/GameCoreTests.swift [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import ComposableArchitecture
+import GameFeature
```
Comment (mbrandonw) [inline Tests/GameCoreTests/GameCoreTests.swift:2]:
```suggestion
import GameCore
```

**Review by luismedicina23 — APPROVED** [review]

**Review by luismedicina23 — APPROVED** [review]

File: Sources/GameCore/GameCore.swift [hunk 2]
```diff
@@ -29,6 +29,7 @@ import UserDefaultsClient
 
 public struct GameState: Equatable {
   public var activeGames: ActiveGamesState
+  public var alert: AlertState<GameAction.AlertAction>?
```
Comment (luismedicina23) [inline Sources/GameCore/GameCore.swift:32]: Sources/ServerConfig/ServerConfig.swift

Comment (luismedicina23) [reply Sources/GameCore/GameCore.swift:32]: > Sources/ServerConfig/ServerConfig.swift

@mbrandonw

**Review by luismedicina23 — APPROVED** [review]

**Review by luismedicina23 — COMMENTED** [review]
@mbrandonw 
Sry 
I'm new and no speek English

**Review by stephencelis — APPROVED** [review]
Looks good!

**Pull request merged**